sycl/test/extensions/annotated_usm/invalid_usm_kind.cpp

```diff
@@ -0,0 +1,113 @@
+  
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_host_annotated'}}
+  TEST(malloc_host_annotated, N, q, properties{usm_kind_shared, alignment<4>});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_host_annotated'}}
+  TEST(malloc_host_annotated, N, Ctx, properties{usm_kind_device});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_host_annotated'}}
+  TEST((malloc_host_annotated<int>), N, q, properties{buffer_location<1>, usm_kind_shared});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_host_annotated'}}
+  TEST((malloc_host_annotated<int>), N, Ctx, properties{buffer_location<2>, usm_kind_device});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_device_annotated'}}
+  TEST(malloc_device_annotated, N, q, properties{usm_kind_host});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_device_annotated'}}
+  TEST(malloc_device_annotated, N, dev, Ctx, properties{usm_kind_shared});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_device_annotated'}}
+  TEST((malloc_device_annotated<int>), N, q, properties{buffer_location<1>, usm_kind_host});
+
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Input property list contains conflicting USM kind.}}
+  // expected-error@+1 {{no matching function for call to 'malloc_device_annotated'}}
+  TEST((malloc_device_annotated<int>), N, dev, Ctx, properties{buffer_location<2>, usm_kind_shared});
+}
+
+// Duplicated properties (consistent or conflicting) exist in the property list
```

Comment:
Please change this comment to "Conflicting usm_kind properties exist in the property list.".

Reply:
"Duplicated properties" means the property key of the two properties are the same.
`usm_kind_device` and `usm_kind_host` share the same property key, so they are "duplicated properties"